### ISSUE TITLE
wiring for 9:1 unun 

### ISSUE THREAD

**Issue by benmarwick** [DG1JAN/UniBalun#7]
Hi Jan,

Thank you so much for making available the plans for these very versatile boards. I am attempting to make the 9:1 unun by following the image here https://github.com/DG1JAN/UniBalun/blob/main/UniBalun_1to9_example.jpg and just want to check I have the wiring correct.

Here's the description of my wiring:
-  start of wire 1 joined to end of wire 2. 
-  start of wire 2 joined to end of wire 3, both soldered to b', which goes to the coax centre pin via a bridge between 1 and 4 on the back of the PCB
- start of wire 3 soldered to a, which goes to coax shield and counterpoise attachment A, via a bridge on the back of the PCB
- end of wire 1 soldered to x, which is antenna attachment B

And here's a simple schematic:

```
w1-start -----------|
w2-start ------|    |
w3-start ------>---->----- a - coax shield & cp via bridge to a'
               |    |
               |    |
w1-end -------->---->----- x - B - antenna 
w2-end -------->----|    
w3-end --------|
               |
               b'
               |
          1 bridge to 4 
               |
            coax pin


```

Does that look right to you?

Thank you so much,

Ben

**Comment by DG1JAN** [comment]
Hello Ben,

I'm sorry its a bit hard for me to read the ASCII Schematic. Please have a look to this document
https://github.com/DG1JAN/UniBalun/blob/main/Bauanleitung_UniBalun_als_1zu9.pdf

Unfortunately I made this Instruction only in German yet, but from the pictures I guess its pretty easy to understand how I did the wiring.

best 73 de Jan

**Comment by benmarwick** [comment]
Thank you for your reply, yes I was looking at this image from your PDF and found it confusing to understand what to do with ends of the wires. 

<img width="291" alt="image" src="https://github.com/user-attachments/assets/5bcec70e-5601-43db-91df-802d59f89eae">

I see red, green and black wires on the left, and the other ends of those wires on the right. But for the ends of the wires in the PDF I only see two red and one black. It is not clear what to do with the two green ends, and the other black end. 

Is it the same as this diagram from [vk6ysf.com](https://vk6ysf.com/unun_9-1.htm)

![image](https://github.com/user-attachments/assets/d895d0d3-08b7-4c4c-8e10-92da9beb8c78)

Thank you so much

**Comment by DG1JAN** [comment]
yes, its the same. sorry for confusing. maybe its also because I dont have wire "ends". I use to wound the UnUn in one turn. Means, making a tab after the first coil and then winding two more rounds. (as you can see in the photos of the pdf)
